tests/ecmp/test_ecmp_sai_value.py

```diff
@@ -45,6 +45,21 @@
 offset_cmd = 'bcmcmd  "dump RTAG7_PORT_BASED_HASH 0 392 OFFSET_ECMP"'
 
 
+@pytest.fixture(scope='module')
+def enable_container_autorestart(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    # Enable autorestart for all features
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    feature_list, _ = duthost.get_feature_status()
+    for feature, status in list(feature_list.items()):
+        if status == 'enabled':
```

Comment:
@vivekverma-arista typo here? when status equals disable, then enable it, right?

Reply:
Hi @ZhaohuiS, this is not a typo, `feature_list, _ = duthost.get_feature_status()` ( line 52 ) this gives us the status of the feature itself ( and not container autorestart ) or in other words the 2nd colunn in this CLI output -
```
admin@ld302:~$ show feature status
Feature         State           AutoRestart     SetOwner
--------------  --------------  --------------  ----------
bgp             enabled         disabled
database        always_enabled  always_enabled
dhcp_relay      enabled         disabled        local
eventd          enabled         disabled
lldp            enabled         disabled
macsec          disabled        disabled        local
mgmt-framework  enabled         disabled
mux             enabled         disabled
nat             disabled        disabled
pmon            enabled         disabled
radv            enabled         disabled
sflow           disabled        disabled
snmp            enabled         disabled
swss            enabled         disabled
syncd           enabled         disabled
teamd           enabled         disabled
telemetry       enabled         disabled
```

So we want to enable container autorestart only for those features which are enabled. For example, MUX is only enabled in dualtor topologies and disabled for T0, T1, etc.

Reply:
@vivekverma-arista that makes sense.